tools/speech_data_explorer/data_explorer.py

```diff
@@ -164,14 +165,16 @@ def load_data(data_filename, disable_caching, vocab=None):
                         num_matches += 1
                 wmr_count += num_matches
 
+            word_rate = 0 if item['duration'] == 0 else round(num_words / item['duration'], 2)
```

Comment:
might be safer if we put
word_rate = round(num_words / item['duration'], 2) if item['duration'] > 0 else 0.0

I somehow feel comparing floats with 0 may not work always.

Reply:
Will update this